src/Dotnet.Script/Program.cs

```diff
@@ -142,7 +145,19 @@ private static Task RunCode(string code, string config, bool debugMode, IEnumera
         private static Task Run(bool debugMode, ScriptContext context)
         {
             var logger = new ScriptLogger(Console.Error, debugMode);
-            var compiler = new ScriptCompiler(logger, new ScriptProjectProvider(new ScriptParser(logger), logger));
+            var runtimeDependencyResolver = new RuntimeDependencyResolver(type => ((level, message) =>
```

Comment:
I think it's probably time to get rid of `ScriptLogger` completely and just use the MS logger library. this is not related to this PR though, so we can do it later

Reply:
Agreed. `Dotnet.Script` should use a logging framework such as Microsoft.Extensions.Logging. The reason for keeping the actual logging implementation out of `Dotnet.Script.DependencyModel` is so that we don't run into trouble with different versions of the logging framework.